dynamic_programming/trapped_water.py

```diff
@@ -0,0 +1,61 @@
+"""
+Given an array of non-negative integers representing an elevation map where
+the width of each bar is 1,this program calculates how much rainwater can be trapped.
+
+Example - height = [0,1,0,2,1,0,1,3,2,1,2,1]
+Output: 6
+This problem can be solved using the concept of "DYNAMIC PROGRAMMING".
+
+We calculate the maximum height of bars on the left and right of every bar in array.
+Then iterate over the width of structure and at each index.
+The amount of water that will be stored is equal to minimum of maximum height of bars
+on both sides minus height of bar at current position.
+"""
+
+
+def trapped_rainwater(height=None) -> int:
```

Comment:
Let's use the plural form with an `s` at then end because this is a list of heights, not a single height.
```suggestion
def trapped_rainwater(heights: list[int]) -> int:
```

Reply:
Sure , thank you for reviewing it